common/web/types/src/util/CompilerInterfaces.ts

```diff
@@ -4,7 +4,7 @@
 export interface CompilerEvent {
```

Comment:
My bad, can we go with compiler-interfaces.ts :grin:

Reply:
done